.github/workflows/push.yaml

```diff
@@ -45,6 +45,11 @@ jobs:
     strategy:
       matrix:
         dockerfile: [ "Dockerfile", "Dockerfile.alpine" ]
```

Comment:
I guess this line must be remove when using `include`

Reply:
It doesn't have to be. `Include` matches on the `dockerfile` key.